sfa_dash/template_globals.py

```diff
@@ -40,7 +40,13 @@
 ALLOWED_QUALITY_FLAGS = {
     'USER FLAGGED': 'USER FLAGGED',
     'NIGHTTIME': 'NIGHTTIME',
-    'CLEARSKY': 'CLEARSKY (GHI Only)',
+    'CLEARSKY': 'CLEARSKY (GHI only)',
+    'LIMITS EXCEEDED': 'LIMITS EXCEEDED',
+    'STALE VALUES': 'STALE VALUES (catches fixed values at nighttime)',
```

Comment:
:/ to me this sounds like only applies at nighttime

maybe this?
```suggestion
    'STALE VALUES': 'STALE VALUES (also catches fixed values at nighttime)',
```

Reply:
or 
```suggestion
    'STALE VALUES': 'STALE VALUES (often includes fixed values at nighttime)',
```

Reply:
or is there a page with descriptions of each flag to just link to and not include anything?

Reply:
https://github.com/SolarArbiter/SolarArbiter.github.io/issues/129#issuecomment-635674054

I'll go with just "includes" instead of "often includes" you object